subsys/bluetooth/mesh/shell/cfg.c

```diff
@@ -153,7 +154,7 @@ static int cmd_get_comp(const struct shell *sh, size_t argc, char *argv[])
 		}
 	}
 
-	if (IS_ENABLED(CONFIG_BT_MESH_COMP_PAGE_1) && page == 1) {
```

Comment:
Why did you remove conditional compilation?

Reply:
I will reinstate it 👍

Reply:
Oh, right. This is implementation on cfg CLI side. As far as I can see `CONFIG_BT_MESH_COMP_PAGE_1` option only applies to/effects the cfg SRV side.